modules/olm-mirroring-catalog.adoc

```diff
@@ -200,6 +200,17 @@ If you used the `--manifests-only` flag during the mirroring process and want to
 ====
 --
 
+. Update the `ImageContentSourcePolicy` object:
++
+[source,terminal]
+----
+$ oc adm catalog mirror <index_image> \
+    <mirror_registry>:<port>/<namespace> \
+    [-a ${REG_CREDS}] \
+    [--manifests-only] \
```

Comment:
The --manifest-only should not be optional.  For this step, we are wanting them to include this so that the generated ImageContentSourcePolicy is defined properly for their mirror registry.